src/pdc-api.ts

```diff
@@ -47,7 +47,15 @@ const usePdcApi = <T>(
      */
   }, [path, params.toString()]);
 
-  return response;
+  useEffect(() => {
+    fetchData()
+      .catch(() => {});
+  }, [fetchData]);
+
+  return [
```

Comment:
I wanted to point out that this is less explicit than an object with named attributes (i.e.  `{response: ..., refresh: ...}`).  That said, react already has this pattern of having hooks return tuples in an array.

I am comfortable with changing to an object if that would feel better.